### ISSUE TITLE
Can't scroll down on page

### ISSUE THREAD

**Issue by esques** [ethereum/ens-registrar-dapp#66]
When scrollin down, page returns automatically to top
cast: http://imgur.com/SD0iaVv

Ubuntu 16.04
Chrome Version 57.0.2987.133 (64-bit)

**Comment by luclu** [comment]
a) Same happens using Mist @`develop`:

![kapture 2017-05-04 at 13 41 58](https://cloud.githubusercontent.com/assets/10135872/25702056/7a58c92a-30cf-11e7-8e28-2136f7be778c.gif)


b) Furthermore here is another scroll related issue (page jumps while scrolling): 

![kapture 2017-05-04 at 13 33 54](https://cloud.githubusercontent.com/assets/10135872/25701825/5845ae8a-30ce-11e7-9d2b-5a2d409e02c0.gif)

**Comment by alexvandesande** [comment]
It used to happen with me, but not anymore, is it still happening?

**Comment by kitsunde** [comment]
@alexvandesande Yes, but it doesn't happen under every circumstance. On this page: https://registrar.ens.domains/#ooooooooooooooooooooooooooooo if the window has a very short height, or just slightly below the full height. It works fine. Somewhere in the middle it doesn't work.
